### PR TITLE
breaking: undefined is no longer a valid value for paths.relative

### PR DESCRIPTION
**Pull request by benmccann**
closes https://github.com/sveltejs/kit/issues/11073

### CODE REVIEW

**Comment by dummdidumm** [comment]
Mhm that's true, nevermind then 👍

File: packages/kit/src/exports/public.d.ts [hunk 1]
```diff
@@ -473,15 +473,18 @@ export interface KitConfig {
 		 */
 		base?: '' | `/${string}`;
 		/**
-		 * Whether to use relative asset paths. By default, if `paths.assets` is not external, SvelteKit will replace `%sveltekit.assets%` with a relative path and use relative paths to reference build artifacts, but `base` and `assets` imported from `$app/paths` will be as specified in your config.
+		 * Whether to use relative asset paths. `true` by default in order to support things like the Internet Archive. `false` may have slight performance advantages and may be preferrable for internal applications.
```
Comment (Rich-Harris) [inline packages/kit/src/exports/public.d.ts:476]: I don't think we need this. 'may have slight performance advantages' seems like bollocks to me (why would it?) and will encourage people to do the less resilient thing

```suggestion
		 * Whether to use relative asset paths.
```

File: packages/kit/src/exports/public.d.ts [hunk 2]
```diff
@@ -473,15 +473,18 @@ export interface KitConfig {
...
 		 *
 		 * If `true`, `base` and `assets` imported from `$app/paths` will be replaced with relative asset paths during server-side rendering, resulting in portable HTML.
```
Comment (Rich-Harris) [inline packages/kit/src/exports/public.d.ts:478]: we should qualify this — obviously making an app truly portable would involve making the server archivable which is oxymoronic

```suggestion
		 * If `true`, `base` and `assets` imported from `$app/paths` will be replaced with relative asset paths during server-side rendering, resulting in more portable HTML.
```

**Review by Rich-Harris — COMMENTED** [review]
small comments on the inline documentation but otherwise LGTM

File: packages/kit/src/exports/public.d.ts (hunk 1 above)
Comment (benmccann) [reply packages/kit/src/exports/public.d.ts:476]: >  'may have slight performance advantages' seems like bollocks to me (why would it?)

There are numerous reasons why it does. Perhaps we can address those though, so I'll remove this for now

**Comment by benmccann** [comment]
I got the tests passing, but I'm quite unsure as to whether my changes to the tests are valid. Please take a look at the changes I made to the test @Rich-Harris and make sure I'm not doing something terribly wrong

**Review by Rich-Harris — APPROVED** [review]

**Pull request merged**